<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanificatorUi</class>
 <widget class="QMainWindow" name="PlanificatorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planificador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="stepArea" native="true">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>2320</width>
      <height>981</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="StepListUI" name="stepList" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QDockWidget" name="messageDisplayDock">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>340</y>
      <width>1920</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>524287</width>
      <height>524287</height>
     </size>
    </property>
    <property name="features">
     <set>QDockWidget::DockWidgetFeature::NoDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::DockWidgetArea::BottomDockWidgetArea</set>
    </property>
    <widget class="MessageDisplayUI" name="messageDisplay_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actioncreate"/>
   <addaction name="separator"/>
   <addaction name="actionOpentask"/>
   <addaction name="separator"/>
   <addaction name="actionClosetask"/>
   <addaction name="separator"/>
   <addaction name="actionExecutetask"/>
   <addaction name="actionList"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <action name="actioncreate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nueva Tarea</string>
   </property>
   <property name="toolTip">
    <string>create task</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpentask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abrir Tarea</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionClosetask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cerrar Tarea</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExecutetask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ejecutar Tarea</string>
   </property>
   <property name="toolTip">
    <string>Execute task</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lista de modelos</string>
   </property>
   <property name="toolTip">
    <string>Muestra la lista de modelos </string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCrearPlot">
   <property name="text">
    <string>Crear Plot</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StepListUI</class>
   <extends>QWidget</extends>
   <header>steplistui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageDisplayUI</class>
   <extends>QWidget</extends>
   <header location="global">messageDisplay/messagedisplayui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
